avaframe/runStandardTestsCom1DFA.py

```diff
@@ -47,6 +47,9 @@
 #  valuesList = ['resistance']
 filterType = 'TAGS'
 valuesList = ['standardTest', 'standardTestSnowGlide']
+# filterType = 'AVANAME'
+# valuesList = ['avaAlr']
```

Comment:
remove?

Reply:
Mh, since I alwasy have to look up in the filterBenchmarks how to specifiy the avaname, I'd like to keep it as here as an example...